docs/data/charts/bars/bars.md

```diff
@@ -61,3 +61,22 @@ Bar charts can be rendered with a horizontal layout by providing the `layout="ho
 If you're using [composition](/x/react-charts/#multiple-charts), you should set the property `layout: 'horizontal'` to each bar series object.
 
 {{"demo": "HorizontalBars.js"}}
+
+## Animation
+
+To skip animation at the creation and update of your chart you can use the `skipAnimation` prop.
```

Comment:
Have you considered naming the prop `disableAnimation`?
Or is it technically incorrect?
I'd wager that the `disable` name would be way more intuitive in this case. 🤔

Reply:
The `skipAnimation` comes from the [`react-spring` notation](https://www.react-spring.dev/docs/utilities/use-reduced-motion#stopping-animations).

```js
Globals.assign({
  skipAnimation: false,
})
```

Not sure about why it would be more intuitive with disable

Reply:
The `skipAnimation` comes from the [`react-spring` notation](https://www.react-spring.dev/docs/utilities/use-reduced-motion#stopping-animations).

```js
Globals.assign({
  skipAnimation: false,
})
```

Not sure about why it would be more intuitive with disable

Reply:
Thank you for clarifying.
Yeah, in such a case, we can go with `skip` instead of `disable`.
I was thinking of the global `MUI` codebase scope, where we usually use `disable` for similar props/cases. 🤔